schemas/skills/samples/complex-skillmanifest-2.1.preview-0.json

```diff
@@ -0,0 +1,184 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/skills/skill-manifest-2.1.preview-0.json",
+    "$id": "SkillBot",
+    "name": "Sample skill definition that can handle multiple types of activities",
+    "version": "1.0",
+    "description": "This is a sample skill definition for multiple activity types",
+    "publisherName": "Microsoft",
+    "privacyUrl": "https://myskill.contoso.com/privacy.html",
+    "copyright": "Copyright (c) Microsoft Corporation. All rights reserved.",
+    "license": "",
+    "iconUrl": "https://myskill.contoso.com/icon.png",
+    "tags": [
+        "sample",
+        "travel",
+        "weather"
+    ],
+    "endpoints": [
+        {
+            "name": "americas",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in the Americas",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "00000000-0000-0000-0000-000000000000"
+        },
+        {
+            "name": "eu",
+            "protocol": "BotFrameworkV3",
+            "description": "Production  endpoint for SkillBot in Europe",
+            "endpointUrl": "http://myskill.contoso.com/api/messages",
+            "msAppId": "11111111-0000-0000-0000-000000000000"
+        }
+    ],
+    "dispatchModels": {
+        "languages": {
+            "en": [
+                {
+                    "id": "SkillLuModel-en",
+                    "name": "SkillBot LU (English)",
+                    "contentType": "application/lu",
+                    "url": "http://sample.com/SkillBot-en.lu",
```

Comment:
so in reality, where should a developer put the lu file? is it hosted together with the skill bot?

Reply:
The developer can chose to put the lu file wherever the feel like it, it could be on the same place where the bot is hosted or somewhere else. In other words, we are not prescribing where these files are. Makes sense?

Reply:
We do have a LUIS URI format (same for speech priming) that would enable interrogation through the authoring API (with an authoring key - now retrievable automatically) to extract utterances. I assume by this the URL is being kept open/flexible so could use what ever you like?

Reply:
We also use the suffix of #{IntentName} on a LU file to enable you to reference specific intents and not expose everything within an LU file. For example this is how we were mapping intent(s) to a given action, in the newly proposed model this is mapped to the skill level not individual actions (which is fine) but wonder how we might enable a user to reference a LU file but only certain intents.  We could impose the LU file provided is cut down to just what's needed for skill purposes from the main LU file but wanted to confirm

```
    "utteranceSources": [
            {
              "locale": "en-us",
              "source": [
                "Calendar#CreateCalendarEntry",
                "Calendar#FindMeetingRoom"
              ]
            }
```